### PR TITLE
Style tweaks and a few more comments

### PR DESCRIPTION
**Pull request by garyb**
Still leaves a lot to be desired, but added a few more laws. They could probably do with some expanding on also, not just having the wordy descriptions.

### CODE REVIEW

File: prelude/README.md [hunk 1]
```diff
@@ -42,9 +41,10 @@ asTypeOf :: forall a. a -> a -> a
 This function returns its first argument, and can be used to assert type equalities.
 This can be useful when types are otherwise ambiguous.
 
-E.g.
-
+For example:
+``` purescript
     main = print $ [] `asTypeOf` [0]
```
Comment (paf31) [inline prelude/README.md]: Indentation

**Comment by paf31** [comment]
:shipit: but just see my one comment.

File: prelude/README.md [hunk 2]
```diff
@@ -42,9 +41,10 @@ asTypeOf :: forall a. a -> a -> a
 This function returns its first argument, and can be used to assert type equalities.
 This can be useful when types are otherwise ambiguous.
 
-E.g.
-
+For example:
```
Comment (michaelficarra) [inline prelude/README.md]: This line isn't needed.

File: prelude/README.md [hunk 3]
```diff
@@ -61,8 +61,7 @@ element `id`, just composable morphisms.
 
 `Semigroupoid`s should obey the following rule:
 
-Association:
-    `forall p q r. p <<< (q <<< r) = (p <<< q) <<< r`
+- Association: `forall p q r. p <<< (q <<< r) = (p <<< q) <<< r`
```
Comment (michaelficarra) [inline prelude/README.md]: Isn't this more commonly referred to as associativity? That's certainly how I've always heard it and how I use it myself.

**Comment by michaelficarra** [comment]
LGTM other than the current comments.

**Comment by garyb** [comment]
I think this may conflict with #913 so I'll wait until that is merged and then revise.

**Comment by paf31** [comment]
@garyb That's merged now.

**Comment by michaelficarra** [comment]
Closed and deleted branch? What happened?

**Comment by garyb** [comment]
I messed up squashing the commit, figured I'd just recreate.

**Comment by paf31** [comment]
This seems to have absorbed the `psc-docs` changes but that should be fine.

I agree with @michaelficarra re: "associativity", but other than that, looks great!

**Comment by michaelficarra** [comment]
@garyb can you link to the replacement PR?

**Comment by garyb** [comment]
#915, it's already merged actually.

**Comment by michaelficarra** [comment]
That's okay, I just wanted it here for posterity.

**Pull request closed without merging**